templates/telemetry/core/core.10.schema.json

```diff
@@ -8,7 +8,16 @@
       "enum": [ 10 ]
     },
     @TELEMETRY_CLIENTID_1_JSON@,
-    @TELEMETRY_CORECOMMON_2_JSON@
+    @TELEMETRY_CORECOMMON_2_JSON@,
```

Comment:
What's the reason for putting these here rather than in coreCommon?  Is it because these are possibly temporary fields?

Reply:
I share Ben's concern here. I think these should be in coreCommon so they're applied to all versions of the core schema. The `telemetry.core` view unions v2 through v10 together and relies on them all being identical.

Reply:
I put them here because they seemed specific to v10 of the core ping. This would certainly break the union between v2 to v10 though.

Reply:
Oh that's true, I didn't consider the union

Reply:
My understanding of the history here is that there never should have been multiple versions. The multiple versions only exist because of a misunderstanding that version has to be bumped for adding fields. We got to v10 and cleared that up, and made the decision to basically ignore the version and have all new fields appear in all versions of the table.